### ISSUE TITLE
docs: Code Editor - update header content example

### ISSUE THREAD

**Issue by kmcfaul** [patternfly/patternfly-react#9935]
"With shortcut menu and main header content" example for Code Editor does not actually pass any header content in, and should be updated to have content.

**Comment by thatblindgeye** [comment]
We should be sure to match the structure Core is using in their example

**Comment by Mash707** [comment]
@thatblindgeye does this involve passing the `headerMainContent` prop with something like "JavaScript Example"?

https://github.com/patternfly/patternfly-react/blob/64381f5a4e6de2673ae008db81cb7fd09c8e3eb4/packages/react-code-editor/src/components/CodeEditor/examples/CodeEditorShortcutMainHeader.tsx#L92-L100

**Comment by thatblindgeye** [comment]
Yep exactly! We could also use "Header main content" like the HTML example does
